src/sentry/db/mixin.py

```diff
@@ -42,10 +48,10 @@ def rename_on_pending_deletion(self, fields=None, extra_fields_to_save=None):
 
         with transaction.atomic():
             self.save(update_fields=fields)
-            OrganizationOption.objects.create(
+            OrganizationOption.objects.update_or_create(
```

Comment:
What's the difference (or is there a difference) between `update_or_create` and `create_or_update`? I've mostly seen the latter but looking now I see the former 3 times.

Reply:
My guess is that `create_or_update()` was built in 2013 before `update_or_create()` was introduced in django 2.0.